modules/retail/pages/retail-install-uyuni.adoc

```diff
@@ -0,0 +1,62 @@
+[[retail-install-uyuni]]
+= Install Uyuni Retail Server with openSUSE
+
+{productname} {smr} Server can be installed on openSUSE.
+
+For general requirements, see xref:installation:uyuni-install-requirements.adoc[].
+
+[NOTE]
+====
+For more information about the latest version and updates of openSUSE Leap, see https://doc.opensuse.org/release-notes/.
+====
+
+
+
+== Install Uyuni on openSUSE Leap
+
+.Procedure: Installing openSUSE Leap with Uyuni
```

Comment:
These two headings should be parallel. I'd also put a single sentence in between them.

Reply:
ok, prepared outside of GH.